### PR TITLE
[#9514] Instructor: Course Edit Page: Show better success message 

### LINKED ISSUE #9514: Instructor: Course Edit Page: Show better success message
**Environment**: 
`master` branch

**Steps to reproduce:**
1. Go to `/web/instructor/courses/edit?courseid=<valid course id>`
2. Edit the course details and click `Save changes`
3. Observe success message:
image

**Ideal result:**
The success message can be improved by showing a simple message instead, similar to what we have in v6:
image

### PR DESCRIPTION
**Pull request by dRadest**
Fixes #9514 

* Changed success message content in src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

**status:** ongoing

### CODE REVIEW

**Comment by dRadest** [comment]
@tanhengyeow thanks! I've changed PR title and description. Will probably have some questions as I continue to work on this.

**Comment by dRadest** [comment]
@tanhengyeow all checks have passed. Is there anything else to do?

File: src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts [hunk 1]
```diff
@@ -368,9 +368,8 @@ export class InstructorCourseEditPageComponent implements OnInit {
     this.courseService.updateCourse(this.courseToEdit.id, {
       courseName: newName,
       timeZone: newTimeZone,
-    }).subscribe((course: Course) => {
-      this.statusMessageService.showSuccessMessage(`Updated course [${course.courseId}] details: `
-          + `Name: ${course.courseName}, Time zone: ${course.timeZone}`);
+    }).subscribe(() => {
+      this.statusMessageService.showSuccessMessage('The course has been updated.');
```
Comment (tanhengyeow) [inline src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts]: The current users are more used to seeing `The course has been edited.` and there's no good enough reason to change the phrasing.

What do you think? :)

Comment (dRadest) [reply src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts]: I agree. I see no reason to unnecessarily temper user experience

**Pull request merged**